spec/services/preserved_object_handler_spec.rb

```diff
@@ -382,7 +382,6 @@
       )
       bad_po_handler = described_class.new(druid, 6, incoming_size, ep)
       allow(bad_po_handler).to receive(:moab_validation_errors).and_return([])
-      # NOTE: #increase_version checks class of object (rspec double != PreservedCopy)
       allow_any_instance_of(PreservedCopy).to receive(:save!).and_raise(ActiveRecord::ActiveRecordError)
```

Comment:
if you are removing the comment, to me it implies we could look to receive save on a specific object.  If that is NOT true, then what's changed is the method in question is no longer #increase_version and thus the comment should stay but the method name should be updated.